service/src/test/java/uk/nhs/adaptors/gp2gp/gpc/GpcDocumentTranslatorTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import uk.nhs.adaptors.gp2gp.common.service.FhirParseService;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+import uk.nhs.adaptors.gp2gp.common.service.TimestampService;
+import uk.nhs.adaptors.gp2gp.ehr.EhrDocumentMapper;
+import uk.nhs.adaptors.gp2gp.utils.ResourceTestFileUtils;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class GpcDocumentTranslatorTest {
+    private static final String TEST_FILE_DIRECTORY = "/ehr/request/fhir/";
+    private static final String INPUT_DIRECTORY = "input/";
+    private static final String OUTPUT_DIRECTORY = "output/";
+    private static final String INPUT_PATH = TEST_FILE_DIRECTORY + INPUT_DIRECTORY;
+    private static final String OUTPUT_PATH = TEST_FILE_DIRECTORY + OUTPUT_DIRECTORY;
+    private static final String BINARY_INPUT_FILE = "test-binary.json";
+    private static final String EXPECTED_MHS_OUTBOUND_REQUEST_FILE = "expected-mhs-outbound-request-payload.json";
```

Comment:
I see the issue with using org.json dependency, this test fixture is incorrect. Refer to the story and the MHS outbound API documentation for the correct format.

Reply:
`
{
    "payload": "<COPC_IN000001UK01...</COPC_IN000001UK01>", // generated from template
    "attachments": [
        {
            "content_type": "text/plain", // determined from GPC access document response
            "is_base64": "True", // constant / always true for GP2GP
            "description": "{{AccessDocument.id}}", // determined from GPC access document response
            "payload": "AAAAGGZ0eXBtcDQyAAAAAW1wNDJtcDQx...g4l2iHjEGQUf/3w==" // from GPC access document response
        }
    ]
}
`
Is this the correct one?

Reply:
looks right now 👍